elasticsearch_dsl/query.py

```diff
@@ -624,7 +624,9 @@ def __init__(
             Literal["multiply", "replace", "sum", "avg", "max", "min"], "DefaultType"
         ] = DEFAULT,
         functions: Union[
-            Sequence["types.FunctionScoreContainer"], Dict[str, Any], "DefaultType"
+            Sequence["types.FunctionScoreContainer"],
+            Sequence[Dict[str, Any]],
```

Comment:
This is a bug that I discovered in the generator for anything with the form `Sequence[types.Something]`, which should be given `Sequence[Dict[str, Any]]` as an alternative. And yes, maybe I should switch `Dict` to `Mapping` as well, and will try if that works at some point.